cmd/cip-mm/main.go

```diff
@@ -24,7 +24,7 @@ import (
 	"github.com/spf13/cobra"
 
 	reg "k8s.io/release/pkg/cip/dockerregistry"
-	"k8s.io/release/pkg/log"
+	"sigs.k8s.io/release-utils/log"
```

Comment:
What's the reason to use `sigs.k8s.io` over `k8s.io`?

Reply:
@saschagrunert -- By default, new repositories would live under kubernetes-sigs with the vanity import `sigs.k8s.io`: https://github.com/kubernetes/community/blob/master/github-management/kubernetes-repositories.md#rules-for-new-repositories

We could request [core repositories](https://github.com/kubernetes/community/blob/master/github-management/kubernetes-repositories.md#core-repositories), but that also requires approval from SIG Architecture and I don't feel that it's strictly necessary for these repos.

Thoughts?

Reply:
Yep, I think then we can stick to sigs.k8s.io.

Reply:
SGTM!